packages/cli/src/index.ts

```diff
@@ -0,0 +1,57 @@
+import yargs, { Argv } from "yargs";
+import * as Commands from "./commands";
+
+const blankBuilder = (_: Argv) => {};
+
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+yargs
+  .command(
+    "generate-config",
+    "Generates a sample config file",
+    blankBuilder,
+    Commands.generateConfig
+  )
+  .command(
+    "install-hook",
+    "Installs gitlint-cli as a git commit-msg hook",
+    blankBuilder,
+    Commands.installHook
+  )
+  .command(
+    "uninstall-hook",
+    "Uninstalls gitlint commit-msg hook",
+    blankBuilder,
+    Commands.uninstallHook
+  )
+  .command(
+    ["lint", "*"],
+    'Lints commits in git repository. Reads last commit message from ".git/COMMIT_EDITMSG" by default',
+    {
+      config: {
+        alias: "c",
+        description: "path to config file",
+        type: "string",
+      },
+      from: {
```

Comment:
The standard way to represent a git commit range is to use dotted range syntax: https://git-scm.com/book/en/v2/Git-Tools-Revision-Selection

Can we support that as well?

Reply:
yes but we can only support the double-dot ranges

Reply:
Sure, I think that's okay.